pkg/registry/core/pod/storage/storage_test.go

```diff
@@ -699,6 +702,48 @@ func TestEtcdCreateBinding(t *testing.T) {
 	}
 }
 
+func TestEtcdUpdateUninitialized(t *testing.T) {
```

Comment:
This test is ok, but something in resttest would be good as well (which tests the strategy).